docs/building-modules/keeper.md

```diff
@@ -68,11 +68,17 @@ func (k Keeper) Set(ctx sdk.Context, key string, value valueType)
 
 and go through the following steps:
 
-1. Retrieve the appropriate store from the `ctx` using the `storeKey`. This is done through the `KVStore(storeKey sdk.StoreKey` method of the `ctx`.
-2. Marhsall `value` to `[]byte` using the codec `cdc`. 
+1. Retrieve the appropriate store from the `ctx` using the `storeKey`. This is done through the `KVStore(storeKey sdk.StoreKey)` method of the `ctx`. Then it's prefered to use the `prefix.Store` to access only the desired limited subset of the store for convenience and safety.
+2. Marshall `value` to `[]byte` using the codec `cdc`. 
```

Comment:
wait, I always thought it was double-L in `marshalling`, but single in `marshal`

Reply:
indeed!